src/Adapter/ContainerBuilder.php

```diff
@@ -57,16 +62,80 @@ public static function getContainer($name, $isDebug)
         }
 
         $container = new SfContainerBuilder();
+
+        $parameters = require _PS_ROOT_DIR_ . '/app/config/parameters.php';
+        foreach ($parameters['parameters'] as $parameter => $value) {
+            $container->setParameter($parameter, $value);
+        }
+        $env = $isDebug ? 'dev' : 'prod';
+        $container->setParameter('kernel.bundles', []);
+        $container->setParameter('kernel.root_dir', _PS_ROOT_DIR_ . '/app/');
+        $container->setParameter('kernel.name', 'app');
+        $container->setParameter('kernel.debug', $isDebug);
+        $container->setParameter('kernel.environment', $env);
+        $container->setParameter('kernel.cache_dir', _PS_CACHE_DIR_);
+
         $container->addCompilerPass(new LegacyCompilerPass());
+
+        $moduleRepository = ModuleRepositoryFactory::getInstance()->getRepository();
+        if (null !== $moduleRepository) {
+            $activeModules = $moduleRepository->getActiveModules();
+            self::addDoctrine($container, $activeModules);
+            self::enableComposerAutoloaderOnModules($activeModules);
+        }
+
         $loader = new YamlFileLoader($container, new FileLocator(__DIR__));
-        $env = $isDebug ? 'dev' : 'prod';
         $servicesPath = _PS_CONFIG_DIR_ . "services/${name}/services_${env}.yml";
         $loader->load($servicesPath);
+
         $container->compile();
 
         $dumper = new PhpDumper($container);
         file_put_contents($file, $dumper->dump(array('class' => $containerName)));
 
         return $container;
     }
+
+    /**
+     * Enable auto loading of module Composer autoloader if needed.
+     * Need to be done as earlier as possible in application lifecycle.
+     *
+     * @param array $modules the list of modules
+     */
+    private static function enableComposerAutoloaderOnModules($modules)
+    {
+        foreach ($modules as $module) {
+            $autoloader = _PS_ROOT_DIR_.'/modules/'.$module.'/vendor/autoload.php';
+
+            if (file_exists($autoloader)) {
+                try {
+                    include_once $autoloader;
+                } catch (\Exception $e) {
+
```

Comment:
We care about exception here, we need to know if something failed. Remove the try catch.

Reply:
alright